libs/components/forms/src/lib/modules/input-box/input-box-errors.component.html

```diff
@@ -19,30 +28,43 @@
 
   <sky-input-box-error
     *ngIf="errors['skyCharacterCounter'] as characterCounterError"
+    class="sky-input-box-error-character-counter"
   >
     {{
       'skyux_input_box_error_character_count'
         | skyLibResources : labelText : characterCounterError.limit
     }}
   </sky-input-box-error>
 
-  <sky-input-box-error *ngIf="errors['skyDate']">
+  <sky-input-box-error
+    *ngIf="errors['skyDate']"
+    class="sky-input-box-error-date"
+  >
     {{ 'skyux_input_box_error_date' | skyLibResources }}
   </sky-input-box-error>
 
-  <sky-input-box-error *ngIf="errors['skyEmail']">
+  <sky-input-box-error
+    *ngIf="errors['skyEmail']"
+    class="sky-input-box-error-email"
+  >
     {{ 'skyux_input_box_error_email' | skyLibResources }}
   </sky-input-box-error>
 
-  <sky-input-box-error *ngIf="errors['skyPhoneField']">
+  <sky-input-box-error
+    *ngIf="errors['skyPhoneField']"
+    class="sky-input-box-error-phone"
+  >
     {{ 'skyux_input_box_error_phone' | skyLibResources }}
   </sky-input-box-error>
 
-  <sky-input-box-error *ngIf="errors['skyTime']">
+  <sky-input-box-error
+    *ngIf="errors['skyTime']"
+    class="sky-input-box-error-time"
```

Comment:
Where are these new CSS classes getting defined? Do they exist for the test?

Reply:
Yes, they are only there for unit tests.